builtin/providers/aws/resource_aws_dhcp_option_set.go

```diff
@@ -0,0 +1,98 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/mitchellh/goamz/ec2"
+)
+
+func resourceAwsDhcpOptionSet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDhcpOptionSetCreate,
+		Delete: resourceAwsDhcpOptionSetDelete,
+
+		"dhcp_options": &schema.Schema{
```

Comment:
No need to nest everything under dhcp_options anymore, since this is its own resource